src/main/java/org/thoughtcrime/securesms/connect/DcHelper.java

```diff
@@ -422,9 +422,10 @@ public static ThreadRecord getThreadRecord(Context context, DcLot summary, DcCha
     long date = summary.getTimestamp();
     int unreadCount = getContext(context).getFreshMsgCount(chatId);
 
+    boolean isEmailThread = chatId > DcChat.DC_CHAT_ID_LAST_SPECIAL && !chat.isDeviceTalk() && !chat.isEncrypted();
```

Comment:
to not show the mail icon for read-only threads, we could check for `can_send()` here. let isEncrypted() return true seems indeed wrong in this case.

or we can rename the function to sth. more UI-based, as showAsEmailChat() (as we did for showPadlock())